service_mesh/service_mesh_install/updating-ossm.adoc

```diff
@@ -0,0 +1,41 @@
+[id="updating-ossm"]
+= Updating {ProductName} from version 1.0.1 to 1.0.2
+include::modules/ossm-document-attributes.adoc[]
+:context: installing-ossm
+toc::[]
+
+Updating {ProductName} requires extra steps before you update OpenShift to version 4.2. 
+You must upgrade {ProductName} to 1.0.2 before upgrading OpenShift 4.1.x to 4.2.
+
+.Prerequisites
+
+* {ProductName} version 1.0.1
+* OpenShift version 4.1
+
+.Procedure
+
+. Configure existing SMCP resource requests by running the following patch command. Replace
+the <smcp namespace> and <smcp name> with your specific names.
```

Comment:
Markup the replaceables, but also use underscores instead of spaces, e.g., `<smcp_namespace>`.

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Reply:
Also consider:

s/patch/`oc patch`/

Reply:
s/names./names:/